### ISSUE TITLE
xTorrenty.org

### ISSUE THREAD

**Issue by PrzemekSkw** [Jackett/Jackett#13864]
### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Is there already a request for your tracker?

- [X] I have checked older issues, open and closed

### Type

~~Semi-Private~~ Private

### Tracker details

```markdown
- Name: xTorrenty
- URL: https://xtorrenty.org/
- Description: xTorrenty is a POLISH Private Torrent Tracker for MOVIES / TV / GENERAL
- Minimum Seeding Time and Minimum Ratio (if Private): none
- Additional Comments: 
Invitation only
```

**Comment by ilike2burnthing** [comment]
Just noticing that after N torrents, or maybe N torrents in X time, users are presented with a captcha challenge when accessing a details page. Nothing we can really do about it, but if an issue is opened about downloads failing, the solution seems to be access the site in browser, open a details page, complete the captcha, and you should be able to download again.

**Comment by garfield69** [comment]
v0.20.2593

**Comment by lukjod** [comment]
Althought it works fine on Jackett after try to add in Sonarr and Radarr it shows categories unsupported.
Here is log from sonarr. Can You please help
[https://gist.github.com/lukjod/a8dc4451b5652b3b1c996f0c3130b0bb](url)

**Comment by ilike2burnthing** [comment]
It's not saying 'categories unsupported', at least not in that log, but rather that when using the categories you've selected, there are no results for the keywordless search (in the case of this indexer, the current year).

The tracker prevents an empty search, and doesn't provide a 'most recent' list in the same layout as their search page, so we use the current year instead. Because we're only a week into 2023, there aren't many results, in fact xTorrenty only has software results.

Basically, you'll just have to wait a bit for '2023' TV and Movie results to be uploaded.

**Comment by lukjod** [comment]
@ilike2burnthing thanks for explenation All clear now

**Comment by PrzemekSkw** [comment]
Hello, thanks for adding that tracker to Jackett. 
As I uderstand I have to wait to add it to Sonarr/Radarr because there's nothing added in 2023?

Regards.

**Comment by garfield69** [comment]
you will not be able to add the indexer to Radarr until at least one movie contains `2023` in its title.

**Comment by PrzemekSkw** [comment]
Ohh, I see they add 2022 etc in title :)
Thanks.

**Comment by garfield69** [comment]
https://xtorrenty.org/ SSL handshake failed Error code 525

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://xtorrenty.org/ has gone private. My credentials are not recognised and registration is now invite only.
@ilike2burnthing  do you still have an account?

**Comment by garfield69** [comment]
v0.22.817

**Comment by ilike2burnthing** [comment]
Yea, my account was removed as well. Signup is open if you have an invitation code from someone on their forum, and I can't be bothered chasing after that.
